Big-Spender/readme.md

```diff
@@ -68,39 +68,48 @@ INSERT YOUR QUERY HERE
 **You:** Then here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+select * from spends where description ilike '%fee%';
 ```
 
 **Farnoosh:** Hi, it's me again. It turns out we also need the transactions that have the expense area of 'Better Hospital Food'. Can you help us with that one?
 
 **You:** No worries. Here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+select s.*
+from spends s
+join expense_areas ea on s.expense_area_id = ea.id
+where ea.expense_area ilike 'Better Hospital Food';
 ```
 
 **Claire:** Great, that's very helpful. How about the total amount spent for each month?
 
 **You:** You can get that by using the GROUP BY clause. Here's the query:
 
 ```sql
-CREATE YOUR QUERY HERE
+select to_char(date,'YYYY-MM') as month from spends group by month;
```

Comment:
Good attempt and nice formatting of the date, however you have partially answered the question. Only one column is returned, but two are expected in your results. Which column is missing?